.vscode/launch.json

```diff
@@ -15,7 +15,7 @@
                 "${file}"
             ],
             "env": {
-                "WRITE_ACTUAL_JSON": "False"
+                "WRITE_ACTUAL": "False"
```

Comment:
@gantrior Shouldn't this be "WRITE_ACTUAL_JSON" ? Or the documentation is wrong?

Reply:
fixed